packages/apps-v5/src/commands/ps/type.js

```diff
@@ -40,28 +40,28 @@ Types: ${cli.color.yellow(formation.map((f) => f.type).join(', '))}`)
 
     formation = sortBy(formation, 'type')
 
-    let dynoTotals = [];
-    let isShowingEcoCostMessage = false;
+    let dynoTotals = []
+    let isShowingEcoCostMessage = false
 
     formation.forEach((d) => {
-      if (d.size === 'Eco' && d.quantity !== 0) {
```

Comment:
I'm curious why the quantity check removed?

Reply:
@mimen since we're just displaying the flat monthly fee if the user does have eco, checking the quantity is redundant.